libs/crypto-key/README.md

```diff
@@ -0,0 +1 @@
+# crypto-key
```

Comment:
suggest having a minimal readme here, since devs who want to bring their own key implementation would use this as a reference

Reply:
Good callout! Added some basic docs. Will update as I start consuming the package with some "how tos"